docs(ins): cop-arm instances (#2428)
scaleway/docs-content: compute/instances/reference-content/cost-optimized.mdx

| Max. Bandwidth   	              | From 100 Mbps  to 6 Gbps                                                      |
| CPU Type   	                    | AMD EPYC 7543 (2,8 GHz)                                                       |
| Resources   	                  | Shared vCPUs                                                                  |
| Sizing   	                      | From 1 to 32 vCPUs <br />From 2 to 128 GiB RAM                                |
| vCPU:RAM ratio                  | Various<br />(1:2, 1:4)                                                       |

## COP-ARM Instances

> An innovative option in the **Cost-Optimized** range are COP-ARM Instances, which are powered by ARM CPUs. These Instances mark a significant step in the world of development and general purpose computing. Their ARM architecture is a key feature, offering an excellent price-performance ratio while maintaining various vCPU to RAM configurations. This ARM CPU design is especially efficient for various standard and cloud-native workloads, ensuring cost-effective operations. Ideal for a wide range of cloud applications, COP-ARM Instances are well-suited for managing medium-to-high-traffic web servers, medium-sized databases, and e-commerce platforms, all while leveraging the unique advantages of the ARM architecture.

Reviewer: ```suggestion
An innovative option in the **Cost-Optimized** range are COP-ARM Instances, which are powered by ARM CPUs. These Instances mark a significant step in the world of development and general purpose computing. Their ARM architecture is a key feature, offering an excellent price-performance ratio while maintaining various vCPU to RAM configurations. This ARM CPU design is especially efficient for various standard and cloud-native workloads, ensuring cost-effective operations. Ideal for a wide range of cloud applications, COP-ARM Instances are well-suited for managing medium-to-high-traffic web servers, medium-sized databases, and e-commerce platforms, all the while leveraging the unique advantages of the ARM architecture.
```

> See below the technical specifications of COP-ARM Instances:

Reviewer: ```suggestion
The table below displays the technical specifications of COP-ARM Instances:
```

| Sizing   	                      | From 1 to 32 vCPUs <br />From 2 to 128 GB RAM                                 |
| vCPU:RAM ratio                  | Various<br />(1:2, 1:4)                                                       |

> ## COPARM1 Instances

Author: ```suggestion
## COP-ARM Instances
```

> An innovative option in the **Cost-Optimized** range are COPARM1 Instances, which are powered by ARM CPUs. These Instances mark a significant step in the realm of Development and General Purpose computing. Their ARM-based architecture is a key feature, offering an excellent price-performance ratio while maintaining various vCPU to RAM configurations. This ARM CPU design is especially efficient for various standard and cloud-native workloads, ensuring cost-effective operations. Ideal for a wide range of cloud applications, COPARM1 Instances are well-suited for managing medium-to-high-traffic web servers, medium-sized databases, and e-commerce platforms, all while leveraging the unique advantages of ARM architecture.

Author: ```suggestion
An innovative option in the **Cost-Optimized** range are COP-ARM Instances, which are powered by ARM CPUs. These Instances mark a significant step in the realm of Development and General Purpose computing. Their ARM-based architecture is a key feature, offering an excellent price-performance ratio while maintaining various vCPU to RAM configurations. This ARM CPU design is especially efficient for various standard and cloud-native workloads, ensuring cost-effective operations. Ideal for a wide range of cloud applications, COP-ARM Instances are well-suited for managing medium-to-high-traffic web servers, medium-sized databases, and e-commerce platforms, all while leveraging the unique advantages of ARM architecture.
```

> An innovative option in the **Cost-Optimized** range are COP-ARM Instances, which are powered by ARM CPUs. These Instances mark a significant step in the realm of Development and General Purpose computing. Their ARM-based architecture is a key feature, offering an excellent price-performance ratio while maintaining various vCPU to RAM configurations. This ARM CPU design is especially efficient for various standard and cloud-native workloads, ensuring cost-effective operations. Ideal for a wide range of cloud applications, COP-ARM Instances are well-suited for managing medium-to-high-traffic web servers, medium-sized databases, and e-commerce platforms, all while leveraging the unique advantages of ARM architecture.

Author: ```suggestion
An innovative option in the **Cost-Optimized** range are COP-ARM Instances, which are powered by ARM CPUs. These Instances mark a significant step in the world of development and general purpose computing. Their ARM-based architecture is a key feature, offering an excellent price-performance ratio while maintaining various vCPU to RAM configurations. This ARM CPU design is especially efficient for various standard and cloud-native workloads, ensuring cost-effective operations. Ideal for a wide range of cloud applications, COP-ARM Instances are well-suited for managing medium-to-high-traffic web servers, medium-sized databases, and e-commerce platforms, all while leveraging the unique advantages of ARM architecture.
```

> An innovative option in the **Cost-Optimized** range are COP-ARM Instances, which are powered by ARM CPUs. These Instances mark a significant step in the world of development and general purpose computing. Their ARM-based architecture is a key feature, offering an excellent price-performance ratio while maintaining various vCPU to RAM configurations. This ARM CPU design is especially efficient for various standard and cloud-native workloads, ensuring cost-effective operations. Ideal for a wide range of cloud applications, COP-ARM Instances are well-suited for managing medium-to-high-traffic web servers, medium-sized databases, and e-commerce platforms, all while leveraging the unique advantages of ARM architecture.

Reviewer 2: ```suggestion
An innovative option in the **Cost-Optimized** range are COP-ARM Instances, which are powered by ARM CPUs. These Instances mark a significant step in the world of development and general purpose computing. Their ARM architecture is a key feature, offering an excellent price-performance ratio while maintaining various vCPU to RAM configurations. This ARM CPU design is especially efficient for various standard and cloud-native workloads, ensuring cost-effective operations. Ideal for a wide range of cloud applications, COP-ARM Instances are well-suited for managing medium-to-high-traffic web servers, medium-sized databases, and e-commerce platforms, all while leveraging the unique advantages of the ARM architecture.
```

Also, wouldn't it be confusing to write it COP-ARM while it is actually COPARM everywhere else?

> See below the technical specifications of COPARM1 Instances:

Author: ```suggestion
See below the technical specifications of COP-ARM Instances:
```

| Range                         	| Cost-Optimized                                                              	|
|:--------------------------------|:------------------------------------------------------------------------------|
| Instance Type   	              | COPARM1                                                             	        |
| Availability Zone   	          | PAR2                                                                          |
| Storage   	                    | Block                                                                         |

> | Max. Bandwidth   	              | From 100 Mbps to 3.2 Gbps                                                     |

Reviewer 2: Shouldn't it be _From 200 Mbps_ ?

Author: I saw 100 Mbps - but maybe my source was outdated. I'll update it.

Author: ```suggestion
| Max. Bandwidth   	              | From 200 Mbps to 3.2 Gbps                                                     |
```

> | CPU Type   	                    | ARM (Ampere Altra Max M128)                                                   |

Author: ```suggestion
| CPU Type   	                    | ARM (Ampere Altra Max M128-30)                                                |
```

| Resources   	                  | Shared vCPUs                                                                  |

> | Sizing   	                      | From 2 to 32 vCPUs <br />From 8 to 128 GB RAM                                |

Reviewer 2: Missed this the first time but, to be precise, RAM sizes are in gibibytes. This has to stay consistent with our other docs though.

```suggestion
| Sizing   	                      | From 2 to 32 vCPUs <br />From 8 to 128 GiB RAM                                |
```

Author: Thank you. I pushed a commit to fix it in the other docs.

> | Sizing   	                      | From 2 to 128 vCPUs <br />From 8 to 128 GB RAM                                |

Reviewer 2: ```suggestion
| Sizing   	                      | From 2 to 32 vCPUs <br />From 8 to 128 GB RAM                                |
```
